installer/windows/build_installer.py

```diff
@@ -233,7 +237,15 @@ def create_wxs_file():
             <Directory Id="ProgramMenuFolder" Name="Programs">
                 <Directory Id="ApplicationProgramsFolder" Name="{MANUFACTURER}">
                     <Component Id="ApplicationShortcutProgramMenu" Guid="e2f5b2a0-0b0a-4b1e-9b0e-9b0e9b0e9b0e">
-                        <Shortcut Id="ApplicationStartMenuShortcut" Name="{PRODUCT_NAME}" Target="[#MassaStationRunScript]" WorkingDirectory="INSTALLDIR" />
+                       <Shortcut
+                            Id="ApplicationStartMenuShortcut"
+                            Name="{PRODUCT_NAME}"
+                            Target="[#MassaStationAppEXE]"
```

Comment:
you changed from MassaStationRunScript to MassaStationAppEXE. Tell me the windows cmd terminal stays open with MassaStationAppEXE.


```suggestion
                            Target="[#MassaStationRunScript]"
```